systemd/caddy.service

```diff
@@ -1,18 +1,21 @@
 [Unit]
 Description=Caddy web server front-end and HTTPS termination for Mastodon
-Requires=mastodon-net-external.service
-After=mastodon-net-external.service
+# On first boot, mastodon-setup will need some time to create mastodon.env used below.
```

Comment:
I just have a general question regarding the `*.service` files: did we consider adding a restart policy?

Reply:
That's a very good point. There are a number of minor items to wrap up after this PR (and the upcoming backup/restore one) are merged. I've created https://github.com/flatcar/flatcar-mastodon/issues/5 to track these.